src/lib.rs

```diff
@@ -109,6 +111,17 @@ impl Client {
         let mut results = FxHashMap::with_capacity_and_hasher(kv.len(), Default::default());
 
         for (key, _) in kv {
+            let kr = key.as_ref();
+            if kr.len() > MAX_KEY_LENGTH {
+                results.insert(
+                    key,
+                    Err(Error::Protocol(Status::Error(ErrorKind::Client(
+                        "Key exceeds maximum length of 250 bytes".to_string(),
```

Comment:
Should we define a custom error kind instead?

Reply:
Updated with a custom `KeyTooLong` entry in `ErrorKind` which replaces the `Client` error.  Name sound fine?